data/mayurstwt.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Mayur Sankhala",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Happy since learned development",
+  "avatar": "https://github.com/mayurstwt.png"
```

Please add a comma `,` here to validate your json. 
```suggestion
  "avatar": "https://github.com/mayurstwt.png",
```

---

thanks @kumarsonsoff3